service/internal/workflow/workflow.go

```diff
@@ -234,3 +238,7 @@ func NewParameters(environmentVariablesString, executeCommand, executeContainerI
 type CreateWorkflowResponse struct {
 	WorkflowName string `json:"workflow_name"`
 }
+
+func txIDGrpcHeader(ctx context.Context) grpc.HeaderCallOption {
+	return grpc.HeaderCallOption{HeaderAddr: &metadata.MD{"X-B3-TraceId=%s": []string{fmt.Sprintf("%s", ctx.Value("X-B3-TraceId"))}}}
```

Comment:
does this work/pass the data as we expect to argo workflows? the key (for `ctx.Value()`) is actually a type, not the string (`"X-B3-TraceId"`)

see: https://play.golang.org/p/w0J57Shpno5

Reply:
Oh ,maybe I broke this when creating the type